arrow-syntax.api

```diff
@@ -0,0 +1,1632 @@
+	public static synthetic fun invoke$default (Lkotlin/jvm/functions/Function16;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Ljava/lang/Object;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;ILjava/lang/Object;)Lkotlin/jvm/functions/Function15;
+	public static synthetic fun invoke$default (Lkotlin/jvm/functions/Function16;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Ljava/lang/Object;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;ILjava/lang/Object;)Lkotlin/jvm/functions/Function15;
+	public static synthetic fun invoke$default (Lkotlin/jvm/functions/Function16;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Ljava/lang/Object;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;ILjava/lang/Object;)Lkotlin/jvm/functions/Function15;
+	public static synthetic fun invoke$default (Lkotlin/jvm/functions/Function16;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Ljava/lang/Object;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;ILjava/lang/Object;)Lkotlin/jvm/functions/Function15;
+	public static synthetic fun invoke$default (Lkotlin/jvm/functions/Function16;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Ljava/l
```

Comment:
I believe this is not useful either in most FP scenarios in Kotlin

Reply:
Already deprecated